tests/integration/targets/ipam_fixed_address/tasks/main.yml

```diff
@@ -0,0 +1,212 @@
+---
+- module_defaults:
+    group/infoblox.bloxone.all:
+      csp_url: "{{ csp_url }}"
+      api_key: "{{ api_key }}"
+
+  block:
+    # Create a random IP space name to avoid conflicts
+    - ansible.builtin.set_fact:
+        name: "test-ip-space-{{ 999999 | random | string }}"
+        tag_value: "site-{{ 999999 | random | string }}"
+
+    # Basic tests for Subnet
+    - name: "Create an IP space"
+      infoblox.bloxone.ipam_ip_space:
+        name: "{{ name }}"
+        state: "present"
+      register: ip_space
+
+    - name: "Create a Subnet"
+      infoblox.bloxone.ipam_subnet:
+        address: "10.0.0.0/24"
+        space: "{{ ip_space.id }}"
+        state: "present"
+      register: subnet
+
+    - name: "Create a Fixed Address"
+      infoblox.bloxone.ipam_fixed_address:
+        address: "10.0.0.1"
+        name: "test_fixed_address_ansible"
+        match_type: "mac"
+        match_value: "00:00:00:00:00:00"
+        ip_space: "{{ ip_space.id }}"
+        state: "present"
+      register: fixed_address
+
+    - name: Get information about the Fixed Address
+      infoblox.bloxone.ipam_fixed_address_info:
+        filters:
+          address: "10.0.0.1"
+          ip_space: "{{ ip_space.id }}"
+      register: fixed_address_info
+
+    - assert:
+        that:
+          - fixed_address is not failed
+          - fixed_address_info.objects | length == 1
+          - fixed_address_info.objects[0].id == fixed_address.id
+          - fixed_address_info.objects[0].address == fixed_address.object.address
+
+    - name: "Create a Fixed Address (idempotent)"
+      infoblox.bloxone.ipam_fixed_address:
+        address: "10.0.0.1"
+        match_type: "mac"
+        match_value: "00:00:00:00:00:00"
+        ip_space: "{{ ip_space.id }}"
+        state: "present"
+      register: fixed_address
+    - assert:
+        that:
+          - fixed_address is not changed
+          - fixed_address is not failed
+
+    - name: "Delete a Fixed Address"
```

Comment:
To maintain consistency , we have a create , update delete cycle in tests where after the "check_mode" and "idempotent" tests , we create the object and update it with additional params and add a delete call at the end.